charts/internal/seed-controlplane/charts/cloud-provider-equinix-metal/templates/configmap-observability.yaml

```diff
@@ -1,55 +0,0 @@
-{{- if not .Values.gep19Monitoring }}
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cloud-controller-manager-observability-config
```

Comment:
You could remove https://github.com/gardener/gardener-extension-provider-equinix-metal/blob/f8986006d99af77fd7f4c99dbed14b2476843255/pkg/controller/controlplane/valuesprovider.go#L51.